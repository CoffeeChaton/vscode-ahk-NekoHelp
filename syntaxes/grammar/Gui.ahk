Gui, Add, UpDown, vMyUpDown Range1-10, 5
; https://www.autohotkey.com/docs/v1/lib/GuiControls.htm#UpDown
Gui, Add, UpDown, vMyUpDown Range1-1000, 5
Gui, Add, UpDown, vMyUpDown Range-50-50, 5
Gui, Add, UpDown, vMyUpDown Range-10--5, 5

; Range(?:-)?\d
; https://www.autohotkey.com/docs/v1/lib/GuiControls.htm#UpDown_Options

Gui, 2: + Minimize
Gui, 2: - Minimize
Gui, 2: -Minimize
Gui, 2: -Minimize
Gui, 2: Add
;    2: color should be variable.other.constant.gui.name.ahk

Gui, guiName: + Minimize
Gui, guiName: - Minimize
Gui, guiName: -Minimize
Gui, guiName: Add
;    guiName: color should be variable.other.constant.gui.name.ahk

; Gui, add https://www.autohotkey.com/docs/v1/lib/GuiControls.htm
Gui, add, Text
Gui, add, Edit
Gui, add, UpDown
Gui, add, Picture
Gui, add, Button
Gui, add, Checkbox
Gui, add, Radio
Gui, add, DropDownList
Gui, add, ComboBox
Gui, add, ListBox
Gui, add, ListView
Gui, add, TreeView
Gui, add, Link
Gui, add, Hotkey
Gui, add, DateTime
Gui, add, MonthCal
Gui, add, Slider
Gui, add, Progress
Gui, add, GroupBox
Gui, add, Tab
Gui, add, StatusBar
Gui, add, ActiveX
Gui, add, Custom