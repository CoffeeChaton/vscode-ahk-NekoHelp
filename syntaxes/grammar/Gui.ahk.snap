>Gui, Add, UpDown, vMyUpDown Range1-10, 5
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#        ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.other.options.ahk keyword.gui.ahk
#                ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#                 ^^^^^^^^^^^ source.ahk meta.command.line.gui.ahk
#                            ^^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.range.ahk keyword.string.gui.range.ahk
#                                 ^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.range.ahk constant.numeric.gui.ahk
#                                  ^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.range.ahk
#                                   ^^ source.ahk meta.command.line.gui.ahk constant.numeric.decimal.ahk
#                                     ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#                                      ^ source.ahk meta.command.line.gui.ahk
#                                       ^ source.ahk meta.command.line.gui.ahk constant.numeric.decimal.ahk
>; https://www.autohotkey.com/docs/v1/lib/GuiControls.htm#UpDown
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>Gui, Add, UpDown, vMyUpDown Range1-1000, 5
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#        ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.other.options.ahk keyword.gui.ahk
#                ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#                 ^^^^^^^^^^^ source.ahk meta.command.line.gui.ahk
#                            ^^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.range.ahk keyword.string.gui.range.ahk
#                                 ^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.range.ahk constant.numeric.gui.ahk
#                                  ^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.range.ahk
#                                   ^^^^ source.ahk meta.command.line.gui.ahk constant.numeric.decimal.ahk
#                                       ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#                                        ^ source.ahk meta.command.line.gui.ahk
#                                         ^ source.ahk meta.command.line.gui.ahk constant.numeric.decimal.ahk
>Gui, Add, UpDown, vMyUpDown Range-50-50, 5
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#        ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.other.options.ahk keyword.gui.ahk
#                ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#                 ^^^^^^^^^^^ source.ahk meta.command.line.gui.ahk
#                            ^^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.range.ahk keyword.string.gui.range.ahk
#                                 ^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.range.ahk
#                                  ^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.range.ahk constant.numeric.gui.ahk
#                                    ^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.range.ahk
#                                     ^^ source.ahk meta.command.line.gui.ahk constant.numeric.decimal.ahk
#                                       ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#                                        ^ source.ahk meta.command.line.gui.ahk
#                                         ^ source.ahk meta.command.line.gui.ahk constant.numeric.decimal.ahk
>Gui, Add, UpDown, vMyUpDown Range-10--5, 5
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#        ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.other.options.ahk keyword.gui.ahk
#                ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#                 ^^^^^^^^^^^ source.ahk meta.command.line.gui.ahk
#                            ^^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.range.ahk keyword.string.gui.range.ahk
#                                 ^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.range.ahk
#                                  ^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.range.ahk constant.numeric.gui.ahk
#                                    ^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.range.ahk
#                                     ^ source.ahk meta.command.line.gui.ahk keyword.operator.arithmetic.ahk
#                                      ^ source.ahk meta.command.line.gui.ahk constant.numeric.decimal.ahk
#                                       ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#                                        ^ source.ahk meta.command.line.gui.ahk
#                                         ^ source.ahk meta.command.line.gui.ahk constant.numeric.decimal.ahk
>
>; Range(?:-)?\d
#^^^^^^^^^^^^^^^ source.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>; https://www.autohotkey.com/docs/v1/lib/GuiControls.htm#UpDown_Options
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>
>Gui, 2: + Minimize
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk variable.other.constant.gui.name.ahk
#        ^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^^^^^ source.ahk meta.command.line.gui.ahk keyword.string.gui.show.ahk
>Gui, 2: - Minimize
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk variable.other.constant.gui.name.ahk
#        ^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^^^^^ source.ahk meta.command.line.gui.ahk keyword.string.gui.show.ahk
>Gui, 2: -Minimize
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk variable.other.constant.gui.name.ahk
#        ^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#         ^^^^^^^^ source.ahk meta.command.line.gui.ahk keyword.string.gui.show.ahk
>Gui, 2: -Minimize
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk variable.other.constant.gui.name.ahk
#        ^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#         ^^^^^^^^ source.ahk meta.command.line.gui.ahk keyword.string.gui.show.ahk
>Gui, 2: Add
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk variable.other.constant.gui.name.ahk
#        ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
>;    2: color should be variable.other.constant.gui.name.ahk
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>
>Gui, guiName: + Minimize
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^^^^^^^ source.ahk meta.command.line.gui.ahk variable.other.constant.gui.name.ahk
#              ^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#               ^ source.ahk meta.command.line.gui.ahk
#                ^^^^^^^^ source.ahk meta.command.line.gui.ahk keyword.string.gui.show.ahk
>Gui, guiName: - Minimize
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^^^^^^^ source.ahk meta.command.line.gui.ahk variable.other.constant.gui.name.ahk
#              ^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#               ^ source.ahk meta.command.line.gui.ahk
#                ^^^^^^^^ source.ahk meta.command.line.gui.ahk keyword.string.gui.show.ahk
>Gui, guiName: -Minimize
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^^^^^^^ source.ahk meta.command.line.gui.ahk variable.other.constant.gui.name.ahk
#              ^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#               ^^^^^^^^ source.ahk meta.command.line.gui.ahk keyword.string.gui.show.ahk
>Gui, guiName: Add
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^^^^^^^ source.ahk meta.command.line.gui.ahk variable.other.constant.gui.name.ahk
#              ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
>;    guiName: color should be variable.other.constant.gui.name.ahk
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>
>; Gui, add https://www.autohotkey.com/docs/v1/lib/GuiControls.htm
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>Gui, add, Text
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#        ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.other.options.ahk keyword.gui.ahk
>Gui, add, Edit
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#        ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.other.options.ahk keyword.gui.ahk
>Gui, add, UpDown
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#        ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.other.options.ahk keyword.gui.ahk
>Gui, add, Picture
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#        ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.other.options.ahk keyword.gui.ahk
>Gui, add, Button
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#        ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.other.options.ahk keyword.gui.ahk
>Gui, add, Checkbox
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#        ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.other.options.ahk keyword.gui.ahk
>Gui, add, Radio
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#        ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.other.options.ahk keyword.gui.ahk
>Gui, add, DropDownList
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#        ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^^^^^^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.other.options.ahk keyword.gui.ahk
>Gui, add, ComboBox
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#        ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.other.options.ahk keyword.gui.ahk
>Gui, add, ListBox
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#        ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.other.options.ahk keyword.gui.ahk
>Gui, add, ListView
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#        ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.other.options.ahk keyword.gui.ahk
>Gui, add, TreeView
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#        ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.other.options.ahk keyword.gui.ahk
>Gui, add, Link
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#        ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.other.options.ahk keyword.gui.ahk
>Gui, add, Hotkey
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#        ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.other.options.ahk keyword.gui.ahk
>Gui, add, DateTime
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#        ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.other.options.ahk keyword.gui.ahk
>Gui, add, MonthCal
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#        ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.other.options.ahk keyword.gui.ahk
>Gui, add, Slider
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#        ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.other.options.ahk keyword.gui.ahk
>Gui, add, Progress
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#        ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.other.options.ahk keyword.gui.ahk
>Gui, add, GroupBox
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#        ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.other.options.ahk keyword.gui.ahk
>Gui, add, Tab
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#        ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.other.options.ahk keyword.gui.ahk
>Gui, add, StatusBar
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#        ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^^^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.other.options.ahk keyword.gui.ahk
>Gui, add, ActiveX
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#        ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.other.options.ahk keyword.gui.ahk
>Gui, add, Custom
#^^^ source.ahk meta.command.line.gui.ahk support.function.gui.command.ahk
#   ^^ source.ahk meta.command.line.gui.ahk
#     ^^^ source.ahk meta.command.line.gui.ahk keyword.command.option.gui.param.ahk
#        ^ source.ahk meta.command.line.gui.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.gui.ahk
#          ^^^^^^ source.ahk meta.command.line.gui.ahk meta.keyword.string.gui.other.options.ahk keyword.gui.ahk